web_fontawesome5/static/src/js/form_renderer.js

```diff
@@ -0,0 +1,55 @@
+odoo.define('web_fontawesome5.FormRenderer', function (require) {
+"use strict";
+
+var FormRenderer = require('web.FormRenderer');
+
+FormRenderer.include({
+
+    _renderStatButton: function (node) {
+        var $button = this._super.apply(this, arguments);
+        if (node.attrs.brand_icon) {
+            $('<div>')
```

Comment:
just asking: wouldn't be better to override widgets' xml templates instead?

Reply:
Stats Buttons are computed directly on Javascript on odoo11.
https://github.com/odoo/odoo/blob/11.0/addons/web/static/src/js/views/form/form_renderer.js#L498
The same applies for standard buttons.